pom.xml

```diff
@@ -689,6 +689,21 @@
                     <version>1.2.1</version>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <compilerId>javac-with-errorprone</compilerId>
```

Comment:
if plexus is the default, then we don't need to include it as a dependency, do we?